shell/renderer/electron_api_service_impl.cc

```diff
@@ -64,7 +64,9 @@ void InvokeIpcCallback(v8::Local<v8::Context> context,
   node::Environment* env = node::Environment::GetCurrent(context);
   std::unique_ptr<node::CallbackScope> callback_scope;
   if (env) {
-    callback_scope.reset(new node::CallbackScope(isolate, ipcNative, {0, 0}));
+    node::async_context async_context = {};
+    callback_scope = std::make_unique<node::CallbackScope>(isolate, ipcNative,
+                                                           async_context);
```

Comment:
Are we sure this is functionally equivalent? Just noticing that `{0, 0}` changed to `{}` and want to make sure a subtle bug doesn't sneak in.

Reply:
Keep an eye on that one

Reply:
the member variables will be default initialized, which is 0 for ints